extensions/ql-vscode/src/query-history.ts

```diff
@@ -135,7 +135,7 @@ class HistoryTreeDataProvider implements QueryHistoryDataProvider {
     return this.current;
   }
 
-  push(item: CompletedQuery): void {
+  pushQuery(item: CompletedQuery): void {
```

Comment:
Need to rename so as not to override the `super.push()` method which is entirely different.